workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationInputs/addUserToGroup.ts

```diff
@@ -0,0 +1,29 @@
+import { AuthenticatedUser } from '../../authenticatedUser';
+
+/**
+ * Request object for AddUserToGroup
+ */
+export interface AddUserToGroupRequest {
+  /**
+   * {@link AuthenticatedUser}
+   */
+  authenticatedUser: AuthenticatedUser;
+  /**
+   * User id associated to user to be added to group
+   */
+  userId: string;
```

Comment:
`AuthenticatedUser` has an id param. Is `userId` different? Or can we remove it?

Reply:
authenticatedUser is used for auditing purposes, the userId added to group isn't the userId requesting the action to do so